index.html

```diff
@@ -227,10 +226,9 @@ <h3 id="contact-info" class="contact_head">I'm always interested in hearing abou
       <ul class="footer_icon">
         <li><a href="https://twitter.com/JoseADelphin"><img src="assets/images/social_media/twitter.svg"
               alt="Twitter icon"></a></li>
-        <li><a href="https://www.linkedin.com/in/josedelphin/"><img src="assets/images/social_media/linkedin.svg"
-              alt="Linkedin icon"></a></li>
-        <li><a href="https://github.com/adelphinsucasa"><img src="assets/images/social_media/github.svg"
-              alt="Github icon"></a>
+        <li><a href="https://www.linkedin.com/in/josedelphin/"><img
+              src="assets/images/social_media/linkedin.svg" alt="Linkedin icon"></a></li>
+        <li><a href="https://github.com/adelphinsucasa"><img src="assets/images/social_media/github.svg" alt="Github icon"></a>
         </li>
       </ul>
       <h4>dalejandro.005@gmail.com</h4>
```

Comment:
- [ ]  :recycle: According to the project requirements, let's make the footer look like the figma desing, by removing the line above the social icons and adjusting the margin and padding properly, justifying the items center as here:


  - Your design:
![lowerYou](https://user-images.githubusercontent.com/97834160/206050105-59f40c32-0514-4a71-8158-2775694ab80f.png)

  - Figma:
![lowerFig](https://user-images.githubusercontent.com/97834160/206050129-784de8f6-404e-4822-9bb0-8adb85fedcf9.png)

Reply:
thanks, working on it